src/ApiClient.ts

```diff
@@ -1,21 +1,23 @@
-import { ArticleItem } from "./Shared";
+import axios from 'axios';
+import { ArticleItem, RegisterUserResponse, User } from './Shared';
+
 class ApiClient {
-  static api_base = "https://api.realworld.io/api/";
-  static getArticles = async (): Promise<ArticleItem[]> => {
-    let result = [];
-    try {
-      const response = await fetch(`${this.api_base}articles`);
-      if (response.status === 200) {
-        const data = await response.json();
-        result = data.articles;
-      } else {
-        throw new Error(response.status.toString());
-      }
-    } catch (e: any) {
-      console.log(e, "error");
-    } finally {
-      return result;
+  static api_base = 'https://api.realworld.io/api/';
+
+  static getArticles = async ():Promise<ArticleItem[]> => {
```

Comment:
Please do not forget to format your code.

Reply:
fixed, thanks